static/src/javascripts/projects/common/modules/onward/related.js

```diff
@@ -66,6 +68,17 @@ define([
                 showCount: false
             }).init();
         } else if (fetchRelated) {
+
+
+            if (ab.isInVariant('RelatedContentDisplayAsRecommendation', 'people-who-read-this-also-read-title')) {
+                var relatedContentSection = document.getElementById('related-content');
+                if (relatedContentSection) {
+                    fastdom.write(function () {
+                        $('.fc-container__header__title', relatedContentSection).textContent = 'people who have read this also read';
```

Comment:
classes that are used from JS should always start with js-<something> so people know to check the js and not refactor them without thinking.  Since this is an AB test don't worry though, but if it's productionised watch that one